generic/sysbench.py.data/sysbench.yaml

```diff
@@ -0,0 +1,10 @@
+max-time: null
+max-request: null
+num-threads: 80
+testsupported: !mux
+  cpu:
+    type: 'cpu'
+    cpu-max-prime: 50000
+  threads:
+    type: 'threads'
+    threads-locks: 1
```

Comment:
there's no like break after the `threads-locks:` please resolve this

Reply:
fixed as per comment